src/scripts/jefferson

```diff
@@ -54,13 +54,6 @@ JFFS2_NODETYPE_XREF = JFFS2_FEATURE_INCOMPAT | JFFS2_NODE_ACCURATE | 9
 def mtd_crc(data):
     return (binascii.crc32(data, -1) ^ -1) & 0xFFFFFFFF
 
-def is_safe_path(basedir, path, follow_symlinks=True):
-    if follow_symlinks:
-        matchpath = os.path.realpath(path)
-    else:
-        matchpath = os.path.abspath(path)
-    return basedir == os.path.commonpath((basedir, matchpath))
-
```

Comment:
Why would you remove this ? It was introduced in https://github.com/sviehb/jefferson/commit/41f654eac078cfec240428be457c054613a80851 to protect againt path traversal through malicious JFFS2.

Reply:
I patched this file based on the old commit, so it doesn't include your new code. I'll close this PR

Reply:
Do you plan on submitting a PR based on the current tree ? Otherwise I can create a PR based on your code fix for version handling. Your call :)

Reply:
That would be better if you create a PR and fix it with some of my code. Because now you are the maintner of jefferson. Your safe patch is incompatible with my "Firmware Analyzer". It reports path traversal errors when I use normal firmware.